docs/CustomApp.md

```diff
@@ -82,7 +82,7 @@ export default ({
 };
 ```
 
-You can use this script as a base and then add your own middleares or enhancers, e.g. to allow store persistence with [redux-persist](https://github.com/rt2zz/redux-persist).
+You can use this script as a base and then add your own middlewares or enhancers, e.g. to allow store persistence with [redux-persist](https://github.com/rt2zz/redux-persist).
```

Comment:
Just adding up one thing on the top of your text-cleaning job (https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/).

```suggestion
You can use this script as a base and then add your own middlewares or enhancers, e.g., to allow store persistence with [redux-persist](https://github.com/rt2zz/redux-persist).
```